integration_tests/e2e/induction/inductionSecurityScenarios.cy.ts

```diff
@@ -13,7 +13,6 @@ context('Security tests for creating and updating Inductions', () => {
     cy.task('stubCiagInductionList')
     cy.task('stubActionPlansList')
     cy.task('getPrisonerById')
-    cy.task('getActionPlan')
```

Comment:
I'm guessing all these calls to `getActionPlan` are being removed from these cypress tests because these tests dont go via the Overview page and therefore don't need to get the action plan?

Reply:
Yeah just attempting to clean it up in the tests so we have a better idea of what actually needs action plan/goals endpoints. I removed them all to see which tests actually needed them and added back as appropriate.